File: app/src/main/java/in/woloo/www/shopping/model/UserGiftCardUpdateResponse.java
package in.woloo.www.shopping.model;

import com.google.gson.annotations.SerializedName;

public class UserGiftCardUpdateResponse {

	@SerializedName("transaction_id")
	private int transaction_id;

	@SerializedName("success")
	private boolean success;

	@SerializedName("message")
	private String message;

	@SerializedName("gift_coins")
	private int giftCoins;

	public int getTransaction_id(){
		return transaction_id;
	}

	public boolean isSuccess(){
		return success;
	}

	public String getMessage(){
		return message;
	}


}

File: app/src/main/java/in/woloo/www/shopping/model/UserGiftCardUpdateResponse.kt
package `in`.woloo.www.shopping.model

import com.google.gson.annotations.SerializedName

class UserGiftCardUpdateResponse {
    @kotlin.jvm.JvmField
	@SerializedName("transaction_id")
    val transaction_id: Int = 0

    @SerializedName("success")
    val isSuccess: Boolean = false

    @SerializedName("message")
    val message: String? = null

    @SerializedName("gift_coins")
    private val giftCoins = 0
}